<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuizView</class>
 <widget class="QDialog" name="QuizView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="styleSheet">
    <string>
      QWidget#QuizView {
        background-color: #0d0d0d;
      }
      QPushButton#btnRandom, QPushButton#btnSelect {
        font-size: 18px;
        border-radius: 12px;
        padding: 20px;
        background-color: #1a1a1a;
        border: 2px solid #333333;
      }
      QPushButton#btnRandom:hover, QPushButton#btnSelect:hover {
        background-color: #252525;
        border-color: #e94560;
      }
    </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     
     <!-- Page 0: Menu -->
     <widget class="QWidget" name="pageMenu">
      <layout class="QVBoxLayout" name="layoutMenu">
        <property name="spacing">
            <number>30</number>
        </property>
        <property name="leftMargin">
            <number>50</number>
        </property>
        <property name="rightMargin">
            <number>50</number>
        </property>
        <property name="topMargin">
            <number>50</number>
        </property>
        <property name="bottomMargin">
            <number>50</number>
        </property>

        <item>
            <widget class="QLabel" name="lblTitle">
                <property name="text"><string>Men√∫ de Problemas</string></property>
                <property name="alignment"><set>Qt::AlignCenter</set></property>
                <property name="styleSheet"><string>font-size: 32px; font-weight: bold; color: #ffffff; margin-bottom: 20px;</string></property>
            </widget>
        </item>
        
        <item>
           <layout class="QHBoxLayout" name="layoutButtons">
              <property name="spacing">
                  <number>30</number>
              </property>
              <item>
                  <widget class="QPushButton" name="btnRandom">
                      <property name="text"><string>üé≤  Problema Aleatorio</string></property>
                      <property name="minimumHeight"><number>150</number></property>
                      <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
                  </widget>
              </item>
              <item>
                  <widget class="QPushButton" name="btnSelect">
                      <property name="text"><string>üìã  Seleccionar de Lista</string></property>
                      <property name="minimumHeight"><number>150</number></property>
                      <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
                  </widget>
              </item>
           </layout>
        </item>
        
        <item>
            <spacer name="spacerMenu"><property name="orientation"><enum>Qt::Vertical</enum></property></spacer>
        </item>

        <item>
            <widget class="QPushButton" name="btnBackToChart">
                <property name="text"><string>‚Ü©Ô∏è  Volver a la Carta</string></property>
                <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
                <property name="class" stdset="0"><string>ghost</string></property>
                <property name="styleSheet"><string>font-size: 16px; color: #aaaaaa; background: transparent; border: none;</string></property>
            </widget>
        </item>
      </layout>
     </widget>

     <!-- Page 1: List -->
     <widget class="QWidget" name="pageList">
       <layout class="QVBoxLayout" name="layoutList">
         <item><widget class="QLabel" name="lblListTitle"><property name="text"><string>Selecciona un problema</string></property><property name="styleSheet"><string>font-size: 24px; font-weight: bold; color: white;</string></property></widget></item>
         <item>
            <layout class="QHBoxLayout" name="layoutListActions">
                <item><widget class="QPushButton" name="btnAddQuestion"><property name="text"><string>‚ûï  Crear Pregunta</string></property><property name="class" stdset="0"><string>primary</string></property><property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property></widget></item>
                <item><spacer name="spacerListAction"><property name="orientation"><enum>Qt::Horizontal</enum></property></spacer></item>
            </layout>
         </item>
         <item><widget class="QListWidget" name="problemList"/></item>
         <item><widget class="QPushButton" name="btnBackFromList"><property name="text"><string>‚Ü©Ô∏è  Volver</string></property><property name="class" stdset="0"><string>ghost</string></property></widget></item>
       </layout>
     </widget>

     <!-- Page 2: Problem (Existing) -->
     <widget class="QWidget" name="pageProblem">
       <layout class="QVBoxLayout" name="layoutProblem">
         <item><widget class="QLabel" name="labelQuestion">
           <property name="text"><string>Pregunta...</string></property>
           <property name="wordWrap"><bool>true</bool></property>
           <property name="styleSheet"><string>font-size: 22px; font-weight: bold; color: white; margin-bottom: 20px;</string></property>
         </widget></item>
         
         <item><widget class="QRadioButton" name="rbAnswer1"><property name="text"><string>Respuesta 1</string></property><property name="styleSheet"><string>padding: 15px; font-size: 16px;</string></property><property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property></widget></item>
         <item><widget class="QRadioButton" name="rbAnswer2"><property name="text"><string>Respuesta 2</string></property><property name="styleSheet"><string>padding: 15px; font-size: 16px;</string></property><property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property></widget></item>
         <item><widget class="QRadioButton" name="rbAnswer3"><property name="text"><string>Respuesta 3</string></property><property name="styleSheet"><string>padding: 15px; font-size: 16px;</string></property><property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property></widget></item>
         <item><widget class="QRadioButton" name="rbAnswer4"><property name="text"><string>Respuesta 4</string></property><property name="styleSheet"><string>padding: 15px; font-size: 16px;</string></property><property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property></widget></item>
         
         <item><widget class="QLabel" name="labelFeedback">
           <property name="text"><string></string></property>
           <property name="alignment"><set>Qt::AlignCenter</set></property>
           <property name="styleSheet"><string>font-size: 18px; font-weight: bold; margin: 20px;</string></property>
         </widget></item>

         <item>
           <layout class="QHBoxLayout" name="layoutActions">
             <item><widget class="QPushButton" name="btnVerify"><property name="text"><string>‚úÖ  Verificar Soluci√≥n</string></property><property name="class" stdset="0"><string>primary</string></property><property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property></widget></item>
             <item><widget class="QPushButton" name="btnBackFromProblem"><property name="text"><string>‚Ü©Ô∏è  Volver al Men√∫</string></property><property name="class" stdset="0"><string>ghost</string></property><property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property></widget></item>
           </layout>
         </item>
         <item><spacer name="spacerProb"><property name="orientation"><enum>Qt::Vertical</enum></property></spacer></item>
       </layout>
     </widget>

     <!-- Page 3: Create Problem -->
     <widget class="QWidget" name="pageCreate">
        <layout class="QVBoxLayout" name="layoutCreate">
            <item><widget class="QLabel" name="lblCreateTitle"><property name="text"><string>Crear Nueva Pregunta</string></property><property name="styleSheet"><string>font-size: 20px; font-weight: bold; color: white; margin-bottom: 20px;</string></property></widget></item>
            
            <item><widget class="QLineEdit" name="editQuestionText"><property name="placeholderText"><string>Enunciado de la pregunta...</string></property></widget></item>
            
            <item><widget class="QLabel" name="lblAnswers"><property name="text"><string>Respuestas (Marca la correcta):</string></property><property name="styleSheet"><string>margin-top: 20px; font-weight: bold;</string></property></widget></item>
            
            <item>
                <layout class="QHBoxLayout" name="layoutAns1">
                    <item><widget class="QRadioButton" name="rbCorrect1"><property name="text"><string/></property></widget></item>
                    <item><widget class="QLineEdit" name="editAnswer1"><property name="placeholderText"><string>Opci√≥n 1</string></property></widget></item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="layoutAns2">
                    <item><widget class="QRadioButton" name="rbCorrect2"><property name="text"><string/></property></widget></item>
                    <item><widget class="QLineEdit" name="editAnswer2"><property name="placeholderText"><string>Opci√≥n 2</string></property></widget></item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="layoutAns3">
                    <item><widget class="QRadioButton" name="rbCorrect3"><property name="text"><string/></property></widget></item>
                    <item><widget class="QLineEdit" name="editAnswer3"><property name="placeholderText"><string>Opci√≥n 3</string></property></widget></item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="layoutAns4">
                    <item><widget class="QRadioButton" name="rbCorrect4"><property name="text"><string/></property></widget></item>
                    <item><widget class="QLineEdit" name="editAnswer4"><property name="placeholderText"><string>Opci√≥n 4</string></property></widget></item>
                </layout>
            </item>

            <item>
                <layout class="QHBoxLayout" name="layoutCreateActions">
                    <item><widget class="QPushButton" name="btnSaveQuestion"><property name="text"><string>üíæ  Guardar Pregunta</string></property><property name="class" stdset="0"><string>primary</string></property><property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property></widget></item>
                    <item><widget class="QPushButton" name="btnCancelCreate"><property name="text"><string>‚ùå  Cancelar</string></property></widget></item>
                </layout>
            </item>
            <item><spacer name="spacerCreate"><property name="orientation"><enum>Qt::Vertical</enum></property></spacer></item>
        </layout>
     </widget>

    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
